_topics/en/soft-fork-activation.md

```diff
@@ -0,0 +1,603 @@
+    92,233,720,368.54277039 BTC.  Bitcoin requires that the output
+    amount be less than or equal to the input amount, but this was
+    checked by adding the output values into a single 64-bit
+    floating-point value which can only hold a maximum value of
+    92,233,720,368.54776 before rolling over to negative numbers,
+    starting with -92,233,720,368.54776.  This meant it looked like the
+    transaction spent a total of -0.1 BTC (negative 0.1 BTC).  This
+    bypassed another rule that forbid individual negative outputs---but
+    not negative aggregate amounts---since it assumed the sum of any set
+    of positive values would also be positive.  This gave the person who
+    created that transaction 184 billion BTC.  The trick could be
+    repeated indefinitely at no cost, allowing production of an
+    unlimited number of bitcoins.  Within hours, a soft fork
+    [fix][overflow fix] restricting outputs to 21 million bitcoins was
+    released in [Bitcoin 0.3.10][].  This did require abandoning the
+    chain with the overflowed transaction---a deliberate consensus
+    failure, but one required to ensure Bitcoin remained useful.
+
+  - **Temporarily fix BDB locks issue (March 2013):** in early 2012,
+    Bitcoin developers [became aware][bitcoin core #925] that trying to
+    make too many changes to the UTXO database (chainstate) at the same
+    time could lead to exceeding one of the default capacity limits of
+    the chainstate's database.  With the small size of Bitcoin blocks at
+    the time, this was only found to occur during block chain
+    reorganizations where transactions from multiple blocks were all
+    processed at the same time.  A simple [solution][bitcoin core #930]
+    was implemented: during a reorganization, only process the
+    transactions from a single block at a time.  Later, some people
+    began [asking miners][hearn soft limit] to raise the optional
+    default block size from 250 kilobytes to something larger.  On 12
+    March 2013, one of those miners produced an [almost 1 megabyte
+    block][andresen 998,081] with over 1,700 transactions---by far the
+    largest block on Bitcoin up to that date---and this was enough to
+    exceed the database capacity on many nodes, causing them to consider
+    the block invalid even though it followed all of Bitcoin's explicit
+    consensus rules.  Complicating the situation, a new version of
+    Bitcoin Core had been released which used a different database
+    engine that didn't contain this limit, and it accepted the large
+    block without problems---producing a consensus failure between the
+    two different node versions.  After quickly analyzing the situation,
+    developers encouraged users to temporarily downgrade to the old
+    version of Bitcoin (the version that rejected the large block) and
+    then to upgrade to an [emergency release of Bitcoin Core][bitcoin
+    core 0.8.1] that implemented an soft fork with a temporary block
```

Comment:
Was it an intentional decision to refer to an "old version of Bitcoin" and and "emergency release of Bitcoin **Core**"? I believe the name Bitcoin Core was only used for version 0.9 onwards.

Reply:
Huh, it was deliberate because I was sure that was in 0.8.0, but the 0.9.0 release notes do indeed contain the rebranding notice.  Thanks for reading so closely!